src/adapter/src/coord/sequencer/inner.rs

```diff
@@ -3181,6 +3191,30 @@ impl Coordinator {
         Ok(Self::send_immediate_rows(rows))
     }
 
+    /// WARNING, ENTERING SPOOKY ZONE 3.0
```

Comment:
@aalexandrov note the new subtlety here; If we switch to the fixed "broken" impl, I believe this goes away

do you have a draft branch for fixing that? I might be able to take a look

Reply:
I don't have a draft branch. I went as far as trying to save the `STACK: OnceLock<Arct<dyn Subscriber + Send + Sync>>` before calling `.init()`. See my comment in #21809.

Reply:
sounds good, will look into this!